packages/cli/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the Zowe CLI package will be documented in this file.
 
 ## Recent Changes
 
+- Enhancement: When copying partitioned data sets that have identical member names, the user is now prompted to confirm before the operation occurs in case of potential data loss. [#2349] (https://github.com/zowe/zowe-cli/issues/2349)
```

Comment:
I know that it takes some time for users to start leveraging new command functionality, but we already released a version that copies one PDS to another without prompting for data loss. This PR changes the default behavior for the copy command so that users are now prompted before the copy can continue.

While this work definitely enhances the behavior of the existing command (and I prefer the new behavior as well), I think we might want to label it as a bug fix to justify the breaking change.

Reply:
I believe we don't need to be concerned about a breaking change in this case - although the PR adding the `--safe-replace` option was merged last week it hasn't been published yet.